pages/report.tsx

```diff
@@ -83,6 +86,28 @@ export default function Report() {
   const { session } = useSessionUser();
   const defaultFormFormValues = useInitialFormValues();
   const submitForm = useSubmitSharkSightingForm();
+  const [ImageFileUuid, setImageFileUuid] = useState<string>();
+
+  const onFileChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const fileList = e.target.files;
+    console.log(fileList);
+    if (!fileList) return;
+    for (let i = 0; i < fileList.length; i++) {
+      if (!fileList[i]) return;
+      const uuid = uuidv4();
+      try {
```

Comment:
We should validate that the file is actually an image using its mimetype before uploading!